prbt_hardware_support/msg/BrakeTestRequired.msg

```diff
@@ -0,0 +1,20 @@
+#
```

Comment:
Why not just use http://docs.ros.org/api/std_msgs/html/msg/Bool.html ?

Reply:
Replaced by the standard type, since we do not see a strong reason.